_includes/configuration/urlconverter-callback.md

```diff
@@ -9,7 +9,7 @@ The format of the converter function is: `URLConverter(url, node, on_save)`, whe
 * on_save is always set to true.
 * name = the attribute name that is being set.
 
-This function should return the converted URL as a string. This option is set to an internal {{site.productname}} function <editor>.[convertURL()]({{ site.baseurl }}/api/tinymce/tinymce.editor/) by default. You may call this function from your extension in order to use the built-in convert options.
+This function should return the converted URL as a string. This option is set to an internal {{site.productname}} function [convertURL()]({{ site.baseurl }}/api/tinymce/tinymce.editor/) by default. You may call this function from your extension in order to use the built-in convert options.
```

Comment:
`<editor>` again was being rendered as an actual element and didn't seem to be adding value anyways, so I just removed it.